web/webkit/src/main/resources/toserve/lift.js

```diff
@@ -397,6 +400,37 @@
       }
     }
 
+    function rehydrateComets() {
+      function onSuccess(html) {
+        var iframe = document.createElement("iframe");
+        var doc = document.createElement("html");
+        iframe.appendChild(doc);
+        doc.innerHTML = html;
```

Comment:
Just a reminder to be **very** careful about where the `html` value in this method comes from. My understanding is that our risk here is pretty minor as `doc` and`iframe` are never attached to the page, but whenever I see it in a PR I gotta mention it.

(Old habits die hard.)

Reply:
Yeah, this is fundamental to the design of this feature. Essentially we reload the page from Lift, and steal all of the comet IDs from the resulting HTML.